pom.xml

```diff
@@ -110,7 +110,7 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<version.java>1.8</version.java>
-		<version.swt>3.105.3</version.swt>
+		<version.swt>3.116.100</version.swt>
 	</properties>
```

Comment:
The module list should contain a reference to your new module `<module>jzy3d-emul-gl-swt</module>` otherwise it is not built with everything.

Reply:
done